sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerProvider.java

```diff
@@ -19,7 +19,7 @@ public class JaegerRemoteSamplerProvider implements ConfigurableSamplerProvider
   static final String SAMPLER_ARG_PROPERTY = "otel.traces.sampler.arg";
   static final String RESOURCE_ATTRIBUTE_SERVICE_NAME_PROPERTY = "service.name";
   private static final String ENDPOINT_KEY = "endpoint";
-  private static final String POLLING_INTERVAL = "pollingInterval";
+  private static final String POLLING_INTERVAL = "pollingIntervalMs";
```

Comment:
Given that this component is stable, we minimally need to support the old version for some deprecation period. Arguably, we should retain support indefinitely, but I think we can avoid this as this is clearly a bug.
```
    String pollingInterval = params.get(POLLING_INTERVAL);
    // TODO: remove support for deprecated pollingInterval after 1.48.0
    if (pollingInterval == null) {
       pollingInterval = params.get("pollingIntervalMs");
       if (pollingInterval != null) {
          logger.warn(SAMPLER_ARG_PROPERTY + " included deprecated pollingInterval argument, which will be removed after 1.48.0. Please use pollingIntervalMs instead"); 
       }
    }
    if (pollingInterval != null) {
      builder.setPollingInterval(Integer.valueOf(pollingInterval), TimeUnit.MILLISECONDS);
    }
```

Reply:
There's no harm in retaining support for the old property name indefinitely. 👍🏽

Reply:
Pushed a commit with this fix. Omitted any indication of removing the deprecated property.